BrowserKit/Sources/ToolbarKit/ToolbarElement.swift

```diff
@@ -32,6 +32,9 @@ public struct ToolbarElement: Equatable {
     /// Indicates if the element has a custom color
     let hasCustomColor: Bool
 
+    /// Indicates if the element has a highlighted color when tapping on it
```

Comment:
Is the highlighted color only used when tapping on it or is it also the indicator that a feature is active?